pom.xml

```diff
@@ -543,7 +543,7 @@
             <dependency>
                 <groupId>io.trino.hive</groupId>
                 <artifactId>hive-apache</artifactId>
-                <version>3.1.2-9</version>
+                <version>3.1.2-11-SNAPSHOT</version>
```

Comment:
remember

Reply:
yeah
Fortunately it will not pass CI in this shape